fzf.el

```diff
@@ -86,11 +101,11 @@
     (split-window-vertically window-height)
     (when fzf/position-bottom (other-window 1))
     (if fzf/args
-        (apply 'make-term "fzf" fzf/executable nil (split-string fzf/args " "))
+        (apply 'make-term "fzf" fzf/executable nil fzf/args)
       (make-term "fzf" fzf/executable))
     (switch-to-buffer buf)
     (linum-mode 0)
-    (set-window-margins nil 1)
+    ;;(set-window-margins nil 1)
```

Comment:
either's fine, since without this change removed your PR wouldn't work anyways, right?